Dollar/Dollar/Dollar.swift

```diff
@@ -276,6 +276,19 @@ public class $ {
         return array
     }
     
+    /// Call the callback on all elements that meet the when condition
+    ///
+    /// :param array The array to check.
+    /// :param callback Check whether element value is true or false.
+    /// :param when Condition to check before performing callback
+    /// :return First element from the array.
+    public class func each<T>(array: [T], when: (T) -> Bool, callback: (T) -> ()) ->[T] {
+        for elem in array where when(elem) {
```

Comment:
Can you do `return $.each(array, when: when, callback: callback)` here so that we can change code in one place and it changes everywhere.